src/views/ProfileMenu/index.jsx

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { withStyles } from '@material-ui/core/styles';
+import AccountCircle from '@material-ui/icons/AccountCircle';
+import {
+  Avatar, IconButton, Menu, MenuItem, Typography,
+} from '@material-ui/core';
+
+const styles = theme => ({
+  root: {
+    display: 'flex',
+  },
+  grow: {
+    marginRight: theme.spacing.unit * 15,
+  },
+  avatar: {
+    width: 25,
+    height: 25,
+    marginRight: theme.spacing.unit * 15,
+  },
+});
+
+class ProfileMenu extends React.Component {
+    state = {
+      anchorEl: null,
+    };
+
+    handleMenu = (event) => {
+      this.setState({ anchorEl: event.currentTarget });
+    };
+
+    handleClose = () => {
+      this.setState({ anchorEl: null });
+    };
+
+    renderAvatar = () => {
+      const { classes, userSession } = this.props;
+      if (userSession.picture) {
```

Comment:
```javascript
return userSession.picture ? (
        <Avatar
            alt={userSession.name}
            src={userSession.picture}
            className={classes.avatar}
          />
  ) : (
     <AccountCircle className={classes.avatar} />
  );
```

Reply:
Thanks, I updated, and because userSession is not in `props` now so I passed it to `renderAvatar` function